### PR TITLE
[GOG] Support launch options

### PR DESCRIPTION
**Pull request by imLinguin**
This PR enables launch options for GOG games like in Legendary, except here we are loading the file every time when we open game page

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by imLinguin** [comment]
Closes #2488

File: src/backend/main.ts [hunk 1]
```diff
@@ -1474,6 +1474,10 @@ ipcMain.handle('syncGOGSaves', async (event, gogSaves, appName, arg) =>
   getGame(appName, 'gog').syncSaves(arg, '', gogSaves)
 )
 
+ipcMain.handle('readGOGGameInfoFile', async (event, appName: string) =>
+  GOGLibrary.get().readInfoFile(appName)
+)
```
Comment (CommandMC) [inline src/backend/main.ts:1479]: This is purely my opinion, but the Frontend just being able to read the whole .info file sounds a little... extreme? I believe we've also had some talks about doing more logic in the Backend in the past

Factoring this in a way that the Frontend just asks for the launch options and the Backend then does that work (reading out the file, filtering out tasks) might be *cleaner*

File: src/frontend/screens/Game/GamePage/index.tsx [hunk 2]
```diff
@@ -158,6 +161,37 @@ export default React.memo(function GamePage(): JSX.Element | null {
...
+                  .then((infoFile) => {
+                    const newLaunchOptions: LaunchOption[] = []
+                    infoFile?.playTasks.forEach((task, index) => {
+                      if (
+                        task.type === 'FileTask' &&
+                        !task?.isHidden &&
+                        task.category !== 'document'
+                      ) {
+                        newLaunchOptions.push({
+                          name: task?.name || infoFile.name,
```
Comment (CommandMC) [inline src/frontend/screens/Game/GamePage/index.tsx:178]: Is it possible for a task's `name` property to be undefined?

If this is correct, could you also adapt the `TaskBase` interface?

File: src/backend/main.ts (hunk 1 above)
Comment (imLinguin) [reply src/backend/main.ts:1479]: I was thinking about it too, and wanted to hear your opinion on that. I'll move it to backend then

**Review by CommandMC — APPROVED** [review]

**Pull request merged**